src/libs/kata-types/src/config/hypervisor/mod.rs

```diff
@@ -1127,6 +1127,14 @@ pub struct Hypervisor {
     /// Vendor customized runtime configuration.
     #[serde(default, flatten)]
     pub vendor: HypervisorVendor,
+
+    /// Disable applying SELinux on the container process.
+    #[serde(default = "yes")]
+    pub disable_guest_selinux: bool,
+}
+
+fn yes() -> bool {
```

Comment:
Is there some place using this method `yes()` ?

Reply:
Yes, it looks like it's not needed to me too.

Reply:
In the commit Pavel says:

> The yes() function is added to comply with serde which seems to insist
> on default values being returned from functions.  Long term, this is
> surely not the best place for this function to live, however given that
> this is currently the first and only place where it's used it seems
> appropriate to put it near its use.  If it ends up being reused elsewhere
> a better place will surely emerge.

Reply:
It does seem like an annoying restriction though!

Reply:
I don't quite understand why serde works this way and I plan to take a closer look to see if it's forced by something, or just an odd design choice.  It is what it is though.  From my research, the name `yes()` seems idiomatic for a function to return `true` (it can't be called `true()` obviously).